apps/media-viewer/src/Mediaviewer.vue

```diff
@@ -207,20 +207,19 @@ export default {
         a: 1
       })
 
+      this.activeMediaFile.path = this.activeMediaFile.path.replace(/^\//, '');
       if (this.publicPage()) {
         const path = [
           '..',
-          'dav',
-          'public-files',
+          'webdav',
           this.activeMediaFile.path
         ].join('/')
 
         return this.$client.files.getFileUrl(path) + '?' + query
       }
       const path = [
         '..',
-        'dav',
```

Comment:
Is the dav endpoint not available on reva/ocis?

Reply:
not yet (soon?)